sway/commands/output/mode.c

```diff
@@ -36,10 +36,6 @@ struct cmd_results *output_cmd_mode(int argc, char **argv) {
 		}
 	} else {
 		// Format is 1234 4321
-		if (!argc) {
```

Comment:
It seems possible to me that this happens.

Reply:
This is checked at the very start of the function, and argc isn't reassigned

Reply:
Ah, we probably need to move this check after `argc--; argv++;`

Reply:
meh, that means I need to check the other warnings like this I ignored because it's just not dead code -_- but righ, that sounds logical.